src/api/procedures/__tests__/subsidizeAccount.ts

```diff
@@ -105,32 +90,52 @@ describe('subsidizeAccount procedure', () => {
       },
     });
 
-    signerToStringStub.withArgs(beneficiaryAccount).returns(address);
+    signerToStringStub.withArgs(beneficiary).returns(address);
 
     const proc = procedureMockUtils.getInstance<SubsidizeAccountParams, AuthorizationRequest>(
       mockContext
     );
 
-    return expect(prepareSubsidizeAccount.call(proc, args)).rejects.toThrow(
+    expect(prepareSubsidizeAccount.call(proc, args)).rejects.toThrow(
       'The Beneficiary Account already has a pending invitation to add this account as a subsidizer'
     );
   });
 
   test('should add an add authorization transaction to the queue', async () => {
-    const randomBeneficiary = entityMockUtils.getAccountInstance({ address: 'someAddress' });
+    const randomBeneficiary = entityMockUtils.getAccountInstance({ address: 'randomAddress' });
```

Comment:
The reason for using separate beneficiary was because I wanted to create two separate authorizations sent by subsidizer (one to the argument beneficiary and one to some random one). 

I have replaced "random" with "mock" for clarity.